processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/core.IM.MI.asm (1 of 5)
processing file asmFiles/core.IM.MS.asm (2 of 5)
processing file asmFiles/core.IS.SS.asm (3 of 5)
processing file asmFiles/core.MM.asm (4 of 5)
processing file asmFiles/core.SM.SI.asm (5 of 5)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/mergesort.asm (1 of 1)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/core.IM.MI.asm (1 of 5)
--- memsim.hex	2024-04-14 18:28:31.979470935 -0400
+++ memcpu.hex	2024-04-14 17:52:37.246912963 -0400
@@ -1,14 +1,33 @@
-:0400000000000005F7
-:040001008C6400000B
-:040002008C6400000A
-:04000300ACA40000A9
-:04000400FFFFFFFFFC
-:04008000340504003F
-:040081008C6200008D
-:04008200ACA200002C
-:04008300FFFFFFFF7D
-:040100003403FF00C5
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000034011FC0E8
+:040001003402008045
+:040002008C4300002B
+:040003002463000171
+:04000400AC43000009
+:04000500C0430000F4
+:04000600246300016E
+:04000700E0430000D2
+:04000800E0430000D1
+:04000900C0430008E8
+:04000A00246300016A
+:04000B00AC430008FA
+:04000C00E0430008C5
+:04000D00AC430008F8
+:04000E00FFFFFFFFF2
+:0400200000000002DA
+:0400240000000002D6
+:0400800034011FC068
+:0400810034020090B5
+:040082008C430000AB
+:0400830024630001F1
+:04008400AC43000089
+:04008500C043000074
+:0400860024630001EE
+:04008700E043000052
+:04008800E043000051
+:04008900C043000868
+:04008A0024630001EA
+:04008B00AC4300087A
+:04008C00E043000845
+:04008D00AC43000878
+:04008E00FFFFFFFF72
 :00000001FF
processing file asmFiles/core.IM.MS.asm (2 of 5)
--- memsim.hex	2024-04-14 18:28:32.193478638 -0400
+++ memcpu.hex	2024-04-14 17:52:37.246912963 -0400
@@ -1,11 +1,33 @@
-:0400000034030400C1
-:0400010034040005BE
-:04000200AC640000EA
-:04000400FFFFFFFFFC
-:040083008C6200008B
-:04008400FFFFFFFF7C
-:0401000000000005F6
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000034011FC0E8
+:040001003402008045
+:040002008C4300002B
+:040003002463000171
+:04000400AC43000009
+:04000500C0430000F4
+:04000600246300016E
+:04000700E0430000D2
+:04000800E0430000D1
+:04000900C0430008E8
+:04000A00246300016A
+:04000B00AC430008FA
+:04000C00E0430008C5
+:04000D00AC430008F8
+:04000E00FFFFFFFFF2
+:0400200000000002DA
+:0400240000000002D6
+:0400800034011FC068
+:0400810034020090B5
+:040082008C430000AB
+:0400830024630001F1
+:04008400AC43000089
+:04008500C043000074
+:0400860024630001EE
+:04008700E043000052
+:04008800E043000051
+:04008900C043000868
+:04008A0024630001EA
+:04008B00AC4300087A
+:04008C00E043000845
+:04008D00AC43000878
+:04008E00FFFFFFFF72
 :00000001FF
processing file asmFiles/core.IS.SS.asm (3 of 5)
--- memsim.hex	2024-04-14 18:28:32.420486809 -0400
+++ memcpu.hex	2024-04-14 17:52:37.246912963 -0400
@@ -1,9 +1,33 @@
-:040000003403FF00C6
-:040001008C6400000B
-:04000300FFFFFFFFFD
-:040082008C6200008C
-:04008300FFFFFFFF7D
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000034011FC0E8
+:040001003402008045
+:040002008C4300002B
+:040003002463000171
+:04000400AC43000009
+:04000500C0430000F4
+:04000600246300016E
+:04000700E0430000D2
+:04000800E0430000D1
+:04000900C0430008E8
+:04000A00246300016A
+:04000B00AC430008FA
+:04000C00E0430008C5
+:04000D00AC430008F8
+:04000E00FFFFFFFFF2
+:0400200000000002DA
+:0400240000000002D6
+:0400800034011FC068
+:0400810034020090B5
+:040082008C430000AB
+:0400830024630001F1
+:04008400AC43000089
+:04008500C043000074
+:0400860024630001EE
+:04008700E043000052
+:04008800E043000051
+:04008900C043000868
+:04008A0024630001EA
+:04008B00AC4300087A
+:04008C00E043000845
+:04008D00AC43000878
+:04008E00FFFFFFFF72
 :00000001FF
processing file asmFiles/core.MM.asm (4 of 5)
--- memsim.hex	2024-04-14 18:28:32.723497716 -0400
+++ memcpu.hex	2024-04-14 17:52:37.246912963 -0400
@@ -1,12 +1,33 @@
-:0400000000000005F7
-:040001008C6400000B
-:04000300ACA40000A9
-:04000400ACA40000A8
-:04000500FFFFFFFFFB
-:04008000340504003F
-:040082008C6200008C
-:04008500FFFFFFFF7B
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000034011FC0E8
+:040001003402008045
+:040002008C4300002B
+:040003002463000171
+:04000400AC43000009
+:04000500C0430000F4
+:04000600246300016E
+:04000700E0430000D2
+:04000800E0430000D1
+:04000900C0430008E8
+:04000A00246300016A
+:04000B00AC430008FA
+:04000C00E0430008C5
+:04000D00AC430008F8
+:04000E00FFFFFFFFF2
+:0400200000000002DA
+:0400240000000002D6
+:0400800034011FC068
+:0400810034020090B5
+:040082008C430000AB
+:0400830024630001F1
+:04008400AC43000089
+:04008500C043000074
+:0400860024630001EE
+:04008700E043000052
+:04008800E043000051
+:04008900C043000868
+:04008A0024630001EA
+:04008B00AC4300087A
+:04008C00E043000845
+:04008D00AC43000878
+:04008E00FFFFFFFF72
 :00000001FF
processing file asmFiles/core.SM.SI.asm (5 of 5)
--- memsim.hex	2024-04-14 18:28:33.055509666 -0400
+++ memcpu.hex	2024-04-14 17:52:37.246912963 -0400
@@ -1,11 +1,33 @@
-:0400000000000005F7
-:040001008C6400000B
-:04000300ACA40000A9
-:04000400FFFFFFFFFC
-:04008000340504003F
-:040082008C6200008C
-:04008400FFFFFFFF7C
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000034011FC0E8
+:040001003402008045
+:040002008C4300002B
+:040003002463000171
+:04000400AC43000009
+:04000500C0430000F4
+:04000600246300016E
+:04000700E0430000D2
+:04000800E0430000D1
+:04000900C0430008E8
+:04000A00246300016A
+:04000B00AC430008FA
+:04000C00E0430008C5
+:04000D00AC430008F8
+:04000E00FFFFFFFFF2
+:0400200000000002DA
+:0400240000000002D6
+:0400800034011FC068
+:0400810034020090B5
+:040082008C430000AB
+:0400830024630001F1
+:04008400AC43000089
+:04008500C043000074
+:0400860024630001EE
+:04008700E043000052
+:04008800E043000051
+:04008900C043000868
+:04008A0024630001EA
+:04008B00AC4300087A
+:04008C00E043000845
+:04008D00AC43000878
+:04008E00FFFFFFFF72
 :00000001FF
processing file asmFiles/core.IM.MI.asm (1 of 5)
processing file asmFiles/core.IM.MS.asm (2 of 5)
processing file asmFiles/core.IS.SS.asm (3 of 5)
processing file asmFiles/core.MM.asm (4 of 5)
processing file asmFiles/core.SM.SI.asm (5 of 5)
processing file asmFiles/test.coherence1.asm (1 of 2)
processing file asmFiles/test.coherence2.asm (2 of 2)
processing file asmFiles/dual.llsc.asm (1 of 2)
processing file asmFiles/dual.mergesort.asm (2 of 2)
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/example.asm (1 of 1)
--- memsim.hex	2024-04-14 19:18:23.726372974 -0400
+++ memcpu.hex	2024-04-14 18:29:18.402141916 -0400
@@ -1,44 +1,231 @@
-:04000000341D3FFC70
-:040001000C00000BE4
-:04000200FFFFFFFFFE
-:04000300C0880000B1
-:040004001500FFFEE6
-:0400050025080001C9
-:04000600E08800008E
-:040007001100FFFBEA
-:0400080003E0000809
-:04000900AC800000C7
-:04000A0003E0000807
-:04000B0027BDFFFC12
-:04000C00AFBF000082
-:04000D003404006057
-:04000E000C000003DF
-:04000F00340A02406D
-:040010008D48000017
-:0400110025090002BB
-:04001200AD490000F4
-:040013003404006051
-:040014000C000009D3
-:040015008FBF000099
-:0400160027BD0004FE
-:0400170003E00008FA
-:04008000341D7FFCB0
-:040081000C000083EC
-:04008200FFFFFFFF7E
-:0400830027BDFFFC9A
-:04008400AFBF00000A
-:0400850034040060DF
-:040086000C00000367
-:04008700340A0240F5
-:040088008D4800009F
-:040089002509000144
-:04008A00AD4900007C
-:04008B0034040060D9
-:04008C000C0000095B
-:04008D008FBF000021
-:04008E0027BD000486
-:04008F0003E0000882
-:040090000000000369
-:040FFE0000000008E7
-:041FFE0000000208D5
+:04000000341EFFFCAF
+:04000100341DFFFCAF
+:0400020034040408B6
+:040003008C10040455
+:0400040034080001BB
+:0400050001102806B8
+:040006000004882545
+:04000700000590253B
+:040008000C00008F59
+:0400090034080001B6
+:04000A00011040069B
+:04000B00020828239C
+:04000C0034090002B1
+:04000D000128400482
+:04000E0034040408AA
+:04000F000088202124
+:04001000000498252B
+:040011000005A02521
+:040012000011202594
+:04001300001228258A
+:040014000013302580
+:040015000014382576
+:0400160034080600A4
+:0400170027BDFFFC06
+:04001800AFA800008D
+:0400190034090400A2
+:04001A008D2A00002B
+:04001B001140FFFE93
+:04001C000C0000A133
+:04001D0027BD0004F7
+:04001E00FFFFFFFFE2
+:04008000341E3FFCEF
+:04008100341D3FFCEF
+:040082003404040836
+:040083008C100404D5
+:04008400340800013B
+:040085000110280638
+:040086003408000238
+:04008700010540042B
+:0400880000882021AB
+:040089000205282321
+:04008A000C00008FD7
+:04008B003408040031
+:04008C003409000132
+:04008D00AD090000B9
+:04008E00FFFFFFFF72
+:04008F00340800042D
+:04009000340900022D
+:0400910001254804F9
+:040092000109082B2D
+:040093001020000C2D
+:040094000088C821F7
+:040095008F380000A0
+:040096001324000629
+:040097008F2FFFFCAC
+:04009800030F082A20
+:040099001020000330
+:04009A00AF2F000084
+:04009B002739FFFC06
+:04009C0008000096C2
+:04009D00AF38000078
+:04009E00250800042D
+:04009F0008000092C3
+:0400A00003E0000871
+:0400A1008FA8000024
+:0400A20014A00002A4
+:0400A30014E0000164
+:0400A400080000C38D
+:0400A50010E0000F58
+:0400A60010A0001591
+:0400A7008C89000040
+:0400A8008CCA0000FE
+:0400A900012A082AF6
+:0400AA00102000051D
+:0400AB00AD0900009B
+:0400AC00250800041F
+:0400AD0024840004A3
+:0400AE0024A5FFFF87
+:0400AF00080000A2A3
+:0400B000AD0A000095
+:0400B100250800041A
+:0400B20024C600045C
+:0400B30024E7FFFF40
+:0400B400080000A29E
+:0400B5008C89000032
+:0400B600AD09000090
+:0400B7002508000414
+:0400B80024A5FFFF7D
+:0400B9002484000497
+:0400BA0010A000088A
+:0400BB00080000B584
+:0400BC008CCA0000EA
+:0400BD00AD0A000088
+:0400BE00250800040D
+:0400BF0024E7FFFF34
+:0400C00024C600044E
+:0400C10010E000014A
+:0400C200080000BC76
+:0400C30003E000084E
+:0401000000000001FA
+:0401010000000040BA
+:0401020000000001F8
+:0401030000000003F5
+:0401040000000004F3
+:0401050000000007EF
+:040106000000000BEA
+:040107000000000CE8
+:0401080000000015DE
+:0401090000000016DC
+:04010A0000000016DB
+:04010B0000000019D7
+:04010C000000001BD4
+:04010D000000001DD1
+:04010E0000000020CD
+:04010F0000000021CB
+:0401100000000029C2
+:0401110000000033B7
+:0401120000000036B3
+:0401130000000038B0
+:040114000000003DAA
+:040115000000003DA9
+:040116000000003EA7
+:0401170000000042A2
+:04011800000000479C
+:040119000000004B97
+:04011A000000004D94
+:04011B000000005090
+:04011C00000000518E
+:04011D00000000518D
+:04011E00000000538A
+:04011F000000005A82
+:040120000000006378
+:040121000000006476
+:0401220000000007D2
+:0401230000000007D1
+:0401240000000007D0
+:0401250000000008CE
+:0401260000000008CD
+:0401270000000008CC
+:0401280000000009CA
+:040129000000000AC8
+:04012A000000000FC2
+:04012B0000000010C0
+:04012C0000000011BE
+:04012D0000000017B7
+:04012E000000001AB3
+:04012F0000000020AC
+:0401300000000020AB
+:0401310000000023A7
+:0401320000000024A5
+:0401330000000026A2
+:0401340000000026A1
+:04013500000000289E
+:040136000000002F96
+:040137000000003094
+:040138000000003291
+:040139000000004181
+:04013A00000000427F
+:04013B00000000437D
+:04013C00000000447B
+:04013D000000005965
+:04013E000000005B62
+:04013F000000005C60
+:040140000000005C5F
+:040141000000006456
+:04018000000000017A
+:040181000000000377
+:040182000000000475
+:040183000000000771
+:040184000000000770
+:04018500000000076F
+:04018600000000076E
+:04018700000000086C
+:04018800000000086B
+:04018900000000086A
+:04018A000000000968
+:04018B000000000A66
+:04018C000000000B64
+:04018D000000000C62
+:04018E000000000F5E
+:04018F00000000105C
+:04019000000000115A
+:040191000000001555
+:040192000000001653
+:040193000000001652
+:040194000000001750
+:04019500000000194D
+:040196000000001A4B
+:040197000000001B49
+:040198000000001D46
+:040199000000002042
+:04019A000000002041
+:04019B000000002040
+:04019C00000000213E
+:04019D00000000233B
+:04019E000000002439
+:04019F000000002636
+:0401A0000000002635
+:0401A1000000002832
+:0401A2000000002930
+:0401A3000000002F29
+:0401A4000000003027
+:0401A5000000003224
+:0401A6000000003322
+:0401A700000000361E
+:0401A800000000381B
+:0401A9000000003D15
+:0401AA000000003D14
+:0401AB000000003E12
+:0401AC00000000410E
+:0401AD00000000420C
+:0401AE00000000420B
+:0401AF000000004309
+:0401B0000000004407
+:0401B1000000004703
+:0401B2000000004BFE
+:0401B3000000004DFB
+:0401B40000000050F7
+:0401B50000000051F5
+:0401B60000000051F4
+:0401B70000000053F1
+:0401B80000000059EA
+:0401B9000000005AE8
+:0401BA000000005BE6
+:0401BB000000005CE4
+:0401BC000000005CE3
+:0401BD0000000063DB
+:0401BE0000000064D9
+:0401BF0000000064D8
+:043FFE0000000600B9
 :00000001FF
processing file asmFiles/example.asm (1 of 1)
processing file asmFiles/add.asm (1 of 45)
--- memsim.hex	2024-04-14 19:19:37.960050874 -0400
+++ memcpu.hex	2024-04-14 19:20:09.816200051 -0400
@@ -1,44 +1,11 @@
-:04000000341D3FFC70
-:040001000C00000BE4
-:04000200FFFFFFFFFE
-:04000300C0880000B1
-:040004001500FFFEE6
-:0400050025080001C9
-:04000600E08800008E
-:040007001100FFFBEA
-:0400080003E0000809
-:04000900AC800000C7
-:04000A0003E0000807
-:04000B0027BDFFFC12
-:04000C00AFBF000082
-:04000D003404006057
-:04000E000C000003DF
-:04000F00340A02406D
-:040010008D48000017
-:0400110025090002BB
-:04001200AD490000F4
-:040013003404006051
-:040014000C000009D3
-:040015008FBF000099
-:0400160027BD0004FE
-:0400170003E00008FA
-:04008000341D7FFCB0
-:040081000C000083EC
-:04008200FFFFFFFF7E
-:0400830027BDFFFC9A
-:04008400AFBF00000A
-:0400850034040060DF
-:040086000C00000367
-:04008700340A0240F5
-:040088008D4800009F
-:040089002509000144
-:04008A00AD4900007C
-:04008B0034040060D9
-:04008C000C0000095B
-:04008D008FBF000021
-:04008E0027BD000486
-:04008F0003E0000882
-:040090000000000369
-:040FFE0000000008E7
-:041FFE0000000208D5
+:0400000000200824B0
+:0400010034210003A3
+:040002000040102486
+:04000300344200067D
+:04000700002218209B
+:040008003404FF08B5
+:04000C00AC830000C1
+:040012008C850000D9
+:04001800FFFFFFFFE8
+:043FC20000000009F2
 :00000001FF
processing file asmFiles/associativity.asm (2 of 45)
processing file asmFiles/branch.asm (3 of 45)
processing file asmFiles/branch.beq.asm (4 of 45)
processing file asmFiles/branch.beq_not_taken.asm (5 of 45)
processing file asmFiles/branch.bne.asm (6 of 45)
processing file asmFiles/branch.bne_not_taken.asm (7 of 45)
processing file asmFiles/core.IM.MI.asm (8 of 45)
processing file asmFiles/core.IM.MS.asm (9 of 45)
processing file asmFiles/core.IS.SS.asm (10 of 45)
processing file asmFiles/core.MM.asm (11 of 45)
processing file asmFiles/core.SM.SI.asm (12 of 45)
processing file asmFiles/count_days.asm (13 of 45)
--- memsim.hex	2024-04-14 19:20:28.854886851 -0400
+++ memcpu.hex	2024-04-14 19:20:31.294974874 -0400
@@ -1,11 +1,47 @@
-:0400000000000005F7
-:040001008C6400000B
-:04000300ACA40000A9
-:04000400FFFFFFFFFC
-:04008000340504003F
-:040082008C6200008C
-:04008400FFFFFFFF7C
-:043FC00000000005F8
-:043FC10000000006F6
-:043FC20000000007F4
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020034010005C0
+:0400030034020001C2
+:04000400340307E8D2
+:040005002042FFFF97
+:040006003407001E9D
+:0400070027BDFFFC16
+:04000800AFA20000A3
+:0400090027BDFFFC14
+:04000A00AFA700009C
+:04000B00001D70253F
+:04000C000C00001BC9
+:04000D008FA70000B9
+:04000E0027BD000406
+:04000F002063F83042
+:040010003405016D45
+:0400110027BDFFFC0C
+:04001200AFA3000098
+:0400130027BDFFFC0A
+:04001400AFA5000094
+:040015000C00001BC0
+:040016008FA60000B1
+:0400170027BD0004FD
+:040018000026882016
+:040019000227082092
+:04001A00FFFFFFFFE6
+:04001B0023AA000410
+:04001C00340CFFFCA5
+:04001D00114C000D75
+:04001E008FA80000A7
+:04001F0027BD0004F5
+:040020008FA90000A4
+:0400210027BD0004F3
+:04002200316B00003E
+:0400230031AD0000FB
+:0400240011A900031B
+:0400250001685820F6
+:0400260021AD000107
+:0400270008000024A9
+:0400280027BDFFFCF5
+:04002900AFAB000079
+:04002A000800001BAF
+:04002B0003E00008E6
+:043FFD000000016D52
+:043FFE000000223865
 :00000001FF
processing file asmFiles/dual.llsc.asm (14 of 45)
processing file asmFiles/dual.mergesort.asm (15 of 45)
processing file asmFiles/example.asm (16 of 45)
processing file asmFiles/fib.asm (17 of 45)
processing file asmFiles/halt2.asm (18 of 45)
processing file asmFiles/halt.asm (19 of 45)
--- memsim.hex	2024-04-14 19:20:38.898249154 -0400
+++ memcpu.hex	2024-04-14 19:20:41.306336023 -0400
@@ -1,5 +1,10 @@
-:040000003401000ABD
-:040001003402FF00C6
-:04000200FFFFFFFFFE
-:04000300AC4100000C
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:040002003401FF00C6
+:040003003402BEAD58
+:04000400AC2200002A
+:04000500FFFFFFFFFB
+:040006003403DEAD34
+:04000700AC23000026
+:043FC0000000BEAD92
 :00000001FF
processing file asmFiles/hazards.control.asm (20 of 45)
processing file asmFiles/hazards.data.asm (21 of 45)
--- memsim.hex	2024-04-14 19:20:43.749424155 -0400
+++ memcpu.hex	2024-04-14 19:20:46.161511169 -0400
@@ -1,13 +1,12 @@
+:0400000000000013E9
 :0400010020030003D5
-:0400020034040006BC
-:040003003407FF00BF
-:04000400104300069F
-:04000500AF820000C6
-:04000600146400037B
-:04000700ACE3000462
-:040008000043282069
-:0400090034060005B4
-:04000A00ACE4000062
-:04000B00FFFFFFFFF5
-:043FC00000000006F7
+:0400020020040004D2
+:0400030020050005CF
+:040004000043102085
+:040005000044102083
+:040006000045102081
+:040007000043102082
+:040008000044102080
+:04000900AF820000C2
+:04000A00FFFFFFFFF6
 :00000001FF
processing file asmFiles/hazards.load.asm (22 of 45)
--- memsim.hex	2024-04-14 19:20:43.749424155 -0400
+++ memcpu.hex	2024-04-14 19:20:48.668601609 -0400
@@ -1,13 +1,9 @@
-:0400010020030003D5
-:0400020034040006BC
-:040003003407FF00BF
-:04000400104300069F
-:04000500AF820000C6
-:04000600146400037B
-:04000700ACE3000462
-:040008000043282069
-:0400090034060005B4
-:04000A00ACE4000062
-:04000B00FFFFFFFFF5
-:043FC00000000006F7
+:04000000340100F0D7
+:040001003402FC10B9
+:040002002063000A6D
+:040003008C2300004A
+:04000400AC43000009
+:04000500FFFFFFFFFB
+:04003C0000000001BF
+:043F040000000001B8
 :00000001FF
processing file asmFiles/hazards.rawar.asm (23 of 45)
--- memsim.hex	2024-04-14 19:20:43.749424155 -0400
+++ memcpu.hex	2024-04-14 19:20:51.128690354 -0400
@@ -1,13 +1,10 @@
-:0400010020030003D5
-:0400020034040006BC
-:040003003407FF00BF
-:04000400104300069F
-:04000500AF820000C6
-:04000600146400037B
-:04000700ACE3000462
-:040008000043282069
-:0400090034060005B4
-:04000A00ACE4000062
-:04000B00FFFFFFFFF5
-:043FC00000000006F7
+:040000003401BEEF1A
+:0400010034050800BA
+:04000200ACA10000AD
+:040003008CA60000C7
+:0400040020C7000110
+:04000500ACA70004A0
+:04000600FFFFFFFFFA
+:040200000000BEEF4D
+:040201000000BEF04B
 :00000001FF
processing file asmFiles/hazards.raw.asm (24 of 45)
--- memsim.hex	2024-04-14 19:20:43.749424155 -0400
+++ memcpu.hex	2024-04-14 19:20:53.557777981 -0400
@@ -1,13 +1,13 @@
-:0400010020030003D5
-:0400020034040006BC
-:040003003407FF00BF
-:04000400104300069F
-:04000500AF820000C6
-:04000600146400037B
-:04000700ACE3000462
-:040008000043282069
-:0400090034060005B4
-:04000A00ACE4000062
-:04000B00FFFFFFFFF5
-:043FC00000000006F7
+:040000003401123481
+:040001003402432161
+:0400020000221820A0
+:040003003404FFFCC6
+:04000400AC830000C9
+:0400050034050800B6
+:04000600ACA10000A9
+:040007008CA60000C3
+:0400080020C700010C
+:04000900FFFFFFFFF7
+:0402000000001234B4
+:043FFF000000555514
 :00000001FF
processing file asmFiles/hazards.rawwarloop.asm (25 of 45)
processing file asmFiles/ihitbeforedhit.asm (26 of 45)
--- memsim.hex	2024-04-14 19:20:54.725820117 -0400
+++ memcpu.hex	2024-04-14 19:20:57.105905976 -0400
@@ -1,49 +1,19 @@
-:040000003401BEEF1A
-:040001003402FEEDDA
-:0400020034040008BA
-:0400030034050800B8
-:04000400ACA10000AB
-:0400050020A500042E
-:04000600ACA20000A8
-:040007008CA60000C3
-:040008008CA70000C1
-:04000900ACA700049C
-:04000A00ACA6000898
-:04000B0020A500101C
-:04000C002084FFFF4E
-:04000D001480FFF666
-:04000E00FFFFFFFFF2
-:04008000FFFFFFFF80
-:040200000000BEEF4D
-:040201000000FEED0E
-:040202000000FEED0D
-:040203000000FEED0C
-:040205000000BEEF48
-:040206000000FEED09
-:040207000000FEED08
-:040208000000FEED07
-:04020A000000BEEF43
-:04020B000000FEED04
-:04020C000000FEED03
-:04020D000000FEED02
-:04020F000000BEEF3E
-:040210000000FEEDFF
-:040211000000FEEDFE
-:040212000000FEEDFD
-:040214000000BEEF39
-:040215000000FEEDFA
-:040216000000FEEDF9
-:040217000000FEEDF8
-:040219000000BEEF34
-:04021A000000FEEDF5
-:04021B000000FEEDF4
-:04021C000000FEEDF3
-:04021E000000BEEF2F
-:04021F000000FEEDF0
-:040220000000FEEDEF
-:040221000000FEEDEE
-:040223000000BEEF2A
-:040224000000FEEDEB
-:040225000000FEEDEA
-:040226000000FEEDE9
+:040000003401FFC008
+:040001003402B00F06
+:040002003405000AB7
+:04000300AC2200002B
+:040004002021FFFCBC
+:040005002063000173
+:040006001465FFFC82
+:04000700FFFFFFFFF9
+:043FE7000000B00F17
+:043FE8000000B00F16
+:043FE9000000B00F15
+:043FEA000000B00F14
+:043FEB000000B00F13
+:043FEC000000B00F12
+:043FED000000B00F11
+:043FEE000000B00F10
+:043FEF000000B00F0F
+:043FF0000000B00F0E
 :00000001FF
processing file asmFiles/ihitdhit.asm (27 of 45)
processing file asmFiles/jump2.asm (28 of 45)
--- memsim.hex	2024-04-14 19:20:59.553994288 -0400
+++ memcpu.hex	2024-04-14 19:21:01.922079714 -0400
@@ -1,10 +1,19 @@
-:0400000034010002C5
-:040001003402FFC006
-:0400020034030003C0
-:04000300AC4300000A
-:040004002021FFFFB9
-:040005002063000173
-:040006001420FFFCC7
-:04000700FFFFFFFFF9
-:043FF00000000004C9
+:04000000340AB00FFF
+:04000100340B0BAD04
+:04000200340CBEEF0D
+:040003003406FF00C0
+:040004000800000AE6
+:04000500ACCB000080
+:040006000C00000CDE
+:04000700ACCA000877
+:04000800ACCC000C70
+:04000900FFFFFFFFF7
+:04000A00ACCA00007C
+:04000B0008000006E3
+:04000C00ACCA000476
+:04000D0008000007E0
+:043FC0000000B00F3E
+:043FC1000000B00F3D
+:043FC2000000B00F3C
+:043FC3000000BEEF4D
 :00000001FF
processing file asmFiles/jump.asm (29 of 45)
--- memsim.hex	2024-04-14 19:20:59.553994288 -0400
+++ memcpu.hex	2024-04-14 19:21:04.397169000 -0400
@@ -1,10 +1,24 @@
-:0400000034010002C5
-:040001003402FFC006
-:0400020034030003C0
-:04000300AC4300000A
-:040004002021FFFFB9
-:040005002063000173
-:040006001420FFFCC7
-:04000700FFFFFFFFF9
-:043FF00000000004C9
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020030210000A9
+:040003003421AAAA50
+:040004003042000086
+:0400050034420BADC9
+:040006003063000063
+:040007003463FF005F
+:040008003405FF08B4
+:040009000C000011D6
+:04000A000060102062
+:04000B00AC610004E0
+:04000C0023E40014D5
+:04000D000080000867
+:04000E00FFFFFFFFF2
+:04000F00AC610008D8
+:04001000FFFFFFFFF0
+:04001100AC7F0000C0
+:040012000800000BD7
+:04001300AC620004D7
+:043FC00000010028D4
+:043FC1000000AAAAA8
+:043FC2000000AAAAA7
 :00000001FF
processing file asmFiles/lwsw.asm (30 of 45)
--- memsim.hex	2024-04-14 19:20:59.553994288 -0400
+++ memcpu.hex	2024-04-14 19:21:06.791255365 -0400
@@ -4,7 +4,9 @@
 :04000300AC4300000A
 :040004002021FFFFB9
 :040005002063000173
-:040006001420FFFCC7
-:04000700FFFFFFFFF9
-:043FF00000000004C9
+:040006002042000490
+:040007001420FFFBC7
+:04000800FFFFFFFFF8
+:043FF00000000003CA
+:043FF10000000004C8
 :00000001FF
processing file asmFiles/mergesort.asm (31 of 45)
processing file asmFiles/mul_procedure.asm (32 of 45)
--- memsim.hex	2024-04-14 19:21:08.542318532 -0400
+++ memcpu.hex	2024-04-14 19:21:11.005407385 -0400
@@ -1,214 +1,40 @@
-:04000000341EFFFCAF
-:04000100341DFFFCAF
-:0400020034040304BB
-:040003008C1003005A
-:0400040034090001BA
-:040005000130280698
-:040006000004882545
-:04000700000590253B
-:040008000C00001DCB
-:0400090034090001B5
-:04000A00013040067B
-:04000B00020828239C
-:04000C0034090002B1
-:04000D000128400482
-:04000E0034040304AF
-:04000F000088202124
-:04001000000498252B
-:040011000005A02521
-:040012000C00001DC1
-:040013000011202593
-:040014000012282589
-:04001500001330257F
-:040016000014382575
-:0400170034080500A4
-:0400180027BDFFFC05
-:04001900AFA800008C
-:04001A000C00002FA7
-:04001B0027BD0004F9
-:04001C00FFFFFFFFE4
-:04001D00340800049F
-:04001E00340A00029E
-:04001F00014548044B
-:040020000109082B9F
-:040021001020000C9F
-:040022000088C82169
-:040023008F38000012
-:04002400132400069B
-:040025008F2FFFFC1E
-:04002600030F082A92
-:0400270010200003A2
-:04002800AF2F0000F6
-:040029002739FFFC78
-:04002A0008000024A6
-:04002B00AF380000EA
-:04002C00250800049F
-:04002D0008000020A7
-:04002E0003E00008E3
-:04002F008FA8000096
-:0400300014A0000216
-:0400310014E00001D6
-:040032000800005171
-:0400330010E0000FCA
-:0400340010A0001503
-:040035008C890000B2
-:040036008CCA000070
-:04003700012A082A68
-:04003800102000058F
-:04003900AD0900000D
-:04003A002508000491
-:04003B002484000415
-:04003C0024A5FFFFF9
-:04003D000800003087
-:04003E00AD0A000007
-:04003F00250800048C
-:0400400024C60004CE
-:0400410024E7FFFFB2
-:040042000800003082
-:040043008C890000A4
-:04004400AD09000002
-:040045002508000486
-:0400460024A5FFFFEF
-:040047002484000409
-:0400480010A00008FC
-:040049000800004368
-:04004A008CCA00005C
-:04004B00AD0A0000FA
-:04004C00250800047F
-:04004D0024E7FFFFA6
-:04004E0024C60004C0
-:04004F0010E00001BC
-:040050000800004A5A
-:0400510003E00008C0
-:04008000FFFFFFFF80
-:0400C00000000040FC
-:0400C100000000013A
-:0400C2000000000337
-:0400C3000000000435
-:0400C4000000000731
-:0400C5000000000B2C
-:0400C6000000000C2A
-:0400C7000000001520
-:0400C800000000161E
-:0400C900000000161D
-:0400CA000000001919
-:0400CB000000001B16
-:0400CC000000001D13
-:0400CD00000000200F
-:0400CE00000000210D
-:0400CF000000002904
-:0400D00000000033F9
-:0400D10000000036F5
-:0400D20000000038F2
-:0400D3000000003DEC
-:0400D4000000003DEB
-:0400D5000000003EE9
-:0400D60000000042E4
-:0400D70000000047DE
-:0400D8000000004BD9
-:0400D9000000004DD6
-:0400DA0000000050D2
-:0400DB0000000051D0
-:0400DC0000000051CF
-:0400DD0000000053CC
-:0400DE000000005AC4
-:0400DF0000000063BA
-:0400E00000000064B8
-:0400E1000000000714
-:0400E2000000000713
-:0400E3000000000712
-:0400E4000000000810
-:0400E500000000080F
-:0400E600000000080E
-:0400E700000000090C
-:0400E8000000000A0A
-:0400E9000000000F04
-:0400EA000000001002
-:0400EB000000001100
-:0400EC0000000017F9
-:0400ED000000001AF5
-:0400EE0000000020EE
-:0400EF0000000020ED
-:0400F00000000023E9
-:0400F10000000024E7
-:0400F20000000026E4
-:0400F30000000026E3
-:0400F40000000028E0
-:0400F5000000002FD8
-:0400F60000000030D6
-:0400F70000000032D3
-:0400F80000000041C3
-:0400F90000000042C1
-:0400FA0000000043BF
-:0400FB0000000044BD
-:0400FC0000000059A7
-:0400FD000000005BA4
-:0400FE000000005CA2
-:0400FF000000005CA1
-:040100000000006497
-:0401400000000001BA
-:0401410000000003B7
-:0401420000000004B5
-:0401430000000007B1
-:0401440000000007B0
-:0401450000000007AF
-:0401460000000007AE
-:0401470000000008AC
-:0401480000000008AB
-:0401490000000008AA
-:04014A0000000009A8
-:04014B000000000AA6
-:04014C000000000BA4
-:04014D000000000CA2
-:04014E000000000F9E
-:04014F00000000109C
-:04015000000000119A
-:040151000000001595
-:040152000000001693
-:040153000000001692
-:040154000000001790
-:04015500000000198D
-:040156000000001A8B
-:040157000000001B89
-:040158000000001D86
-:040159000000002082
-:04015A000000002081
-:04015B000000002080
-:04015C00000000217E
-:04015D00000000237B
-:04015E000000002479
-:04015F000000002676
-:040160000000002675
-:040161000000002872
-:040162000000002970
-:040163000000002F69
-:040164000000003067
-:040165000000003264
-:040166000000003362
-:04016700000000365E
-:04016800000000385B
-:040169000000003D55
-:04016A000000003D54
-:04016B000000003E52
-:04016C00000000414E
-:04016D00000000424C
-:04016E00000000424B
-:04016F000000004349
-:040170000000004447
-:040171000000004743
-:040172000000004B3E
-:040173000000004D3B
-:040174000000005037
-:040175000000005135
-:040176000000005134
-:040177000000005331
-:04017800000000592A
-:040179000000005A28
-:04017A000000005B26
-:04017B000000005C24
-:04017C000000005C23
-:04017D00000000631B
-:04017E000000006419
-:04017F000000006418
-:043FFE0000000500BA
+:0400000033BD00000C
+:0400010037BDFFFC0C
+:0400020031080000C1
+:0400030035080001BB
+:04000400312900009E
+:040005003529000297
+:04000600340E0002B2
+:04000700340F0001B1
+:0400080027BDFFFC15
+:04000900AFA800009C
+:04000A0027BDFFFC13
+:04000B00AFA9000099
+:04000C0027BDFFFC11
+:04000D00AFAE000092
+:04000E0027BDFFFC0F
+:04000F00AFAF00008F
+:040010000C000012CE
+:04001100FFFFFFFFEF
+:0400120023AA000419
+:04001300340CFFFCAE
+:04001400114C000D7E
+:040015008FA80000B0
+:0400160027BD0004FE
+:040017008FA90000AD
+:0400180027BD0004FC
+:04001900316B000047
+:04001A0031AD000004
+:04001B0011A9000324
+:04001C0001685820FF
+:04001D0021AD000110
+:04001E000800001BBB
+:04001F0027BDFFFCFE
+:04002000AFAB000082
+:0400210008000012C1
+:0400220003E00008EF
+:043FFB0000000001C1
+:043FFC0000000002BF
+:043FFD0000000004BC
+:043FFE0000000004BB
 :00000001FF
processing file asmFiles/mult.asm (33 of 45)
processing file asmFiles/search.asm (34 of 45)
--- memsim.hex	2024-04-14 19:21:13.459495914 -0400
+++ memcpu.hex	2024-04-14 19:21:15.936585272 -0400
@@ -1,17 +1,120 @@
-:0400000033BD00000C
-:0400010037BDFFFC0C
-:0400020031080000C1
-:04000300312900009F
-:0400040035080003B8
-:040005003529000297
-:04000600314A00007B
-:04000700316B000059
-:040008001149000397
-:040009000168582012
-:04000A00214A000186
-:04000B0008000008E1
-:04000C0027BDFFFC11
-:04000D00AFAB000095
-:04000E00FFFFFFFFF2
-:043FFE0000000006B9
+:04000000341D00802B
+:0400010034010001C5
+:0400020034020004C0
+:04000300AFA00000AA
+:040004008FA30004C2
+:040005008FA40008BC
+:0400060027A5000C1E
+:040007008CA60000C3
+:0400080000C33823D6
+:0400090010E00004FF
+:04000A0000A2282107
+:04000B00008120232D
+:04000C00108000025E
+:04000D001000FFF9E7
+:04000E00AFA500009A
+:04000F00FFFFFFFFF1
+:0400200000000124B7
+:0400210000005C6F10
+:040022000000006476
+:040023000000087D54
+:0400240000005FCBAE
+:040025000000A41A19
+:04002600000041098C
+:04002700000045226E
+:040028000000700F55
+:040029000000766DF0
+:04002A0000006F6003
+:04002B0000008A5EE9
+:04002C0000009580BB
+:04002D00000070A3BC
+:04002E000000AEA977
+:04002F000000711A42
+:0400300000006F81DC
+:0400310000008F9AA2
+:040032000000258421
+:040033000000A5998B
+:040034000000401573
+:040035000000CE8178
+:040036000000F55B76
+:040037000000399EEE
+:040038000000A23FE3
+:040039000000358806
+:04003A00000033ACE3
+:04003B000000BCE71E
+:04003C0000002A6B2B
+:04003D0000009FA17F
+:04003E000000C94BAA
+:04003F000000C65B9C
+:040040000000006854
+:040041000000F4992E
+:0400420000005F71EA
+:040043000000D06F7A
+:04004400000014DFC5
+:040045000000116541
+:040046000000F88D31
+:0400470000004BA4C6
+:0400480000002E7412
+:0400490000005C6FE8
+:04004A000000D11EC3
+:04004B0000009222FD
+:04004C000000ACDB29
+:04004D000000103867
+:04004E000000AB17EC
+:04004F000000F7CEE8
+:0400500000008A9E84
+:0400510000009AA36E
+:040052000000B49561
+:0400530000008A5EC1
+:040054000000D85977
+:0400550000000BACF0
+:040056000000D0DBFB
+:04005700000035521E
+:040058000000A6B04E
+:040059000000727FB2
+:04005A00000028E496
+:04005B000000E5CFED
+:04005C000000163C4E
+:04005D00000034115A
+:04005E0000008F0708
+:04005F000000FAB7EC
+:0400600000000F3459
+:040061000000DABF02
+:0400620000006F6FBC
+:040063000000C5983C
+:040064000000F4960E
+:0400650000009A9A63
+:040066000000BD6A6F
+:04006700000021363E
+:040068000000810A09
+:040069000000CA5574
+:04006A0000008BCE39
+:04006B0000002AC4A3
+:04006C000000DDCEE5
+:04006D000000DD06AC
+:04006E000000C4FCCE
+:04006F000000FB2F63
+:040070000000EE5F3F
+:040071000000FD305E
+:040072000000C54085
+:040073000000D5F1C3
+:040074000000BDAD1E
+:04007500000045C37F
+:040076000000708A8C
+:040077000000A35989
+:040078000000F40D83
+:040079000000BA06C3
+:04007A000000BACEFA
+:04007B000000B44786
+:04007C0000003F48F9
+:04007D000000899E58
+:04007E00000080847A
+:04007F000000BDB907
+:040080000000A05A82
+:040081000000E22574
+:040082000000FB0C73
+:040083000000B2B215
+:040084000000A4DBF9
+:0400850000008BF9F3
+:04008600000012F76D
 :00000001FF
processing file asmFiles/slt.asm (35 of 45)
processing file asmFiles/subroutine_crc.asm (36 of 45)
processing file asmFiles/test.branchjump1.asm (1 of 7)
--- memsim.hex	2024-04-14 19:21:54.441974884 -0400
+++ memcpu.hex	2024-04-14 19:21:54.423974234 -0400
@@ -13,6 +13,6 @@
 :04000C00AC41000CF7
 :04000D0017E3FFF9FD
 :04000E00FFFFFFFFF2
-:0400220000000018C2
+:0400220000010018C1
 :0400240000000030A8
 :00000001FF
processing file asmFiles/test.coherence1.asm (2 of 7)
processing file asmFiles/test.coherence2.asm (3 of 7)
processing file asmFiles/test.ldst1.asm (4 of 7)
processing file asmFiles/test.ldst2.asm (5 of 7)
--- memsim.hex	2024-04-14 19:21:59.150144838 -0400
+++ memcpu.hex	2024-04-14 19:22:00.304186497 -0400
@@ -1,21 +1,20 @@
-:04000000340100F0D7
-:0400010034020100C4
-:0400020034030200C1
-:0400030034040300BE
-:0400040034050400BB
-:040005008C26000045
-:040006008C2700043F
-:040007008C28000839
-:0400080034040500B7
-:0400090034050600B4
-:04000A00AC46000000
-:04000B00AC470004FA
-:04000C00AC480008F4
-:04000D00FFFFFFFFF3
-:04003C000000733716
-:04003D000000270197
-:04003E000000133774
-:040040000000733712
-:040041000000270193
-:040042000000133770
+:040000003C0AFEEDCB
+:0400010034010F00B7
+:0400020034020800BC
+:04000300354ABEEFCD
+:040005008C23000048
+:040006008C24000442
+:040007008C2500083C
+:04000A00AC43000003
+:04000B00AC440004FD
+:04000C00AC450008F7
+:04000D00AC4A000CED
+:04000E00FFFFFFFFF2
+:040200000000733750
+:0402010000002701D1
+:0402020000001337AE
+:04020300FEEDBEEF5F
+:0403C000000073378F
+:0403C1000000270110
+:0403C20000001337ED
 :00000001FF
processing file asmFiles/test.loadstore.asm (6 of 7)
processing file asmFiles/test.rtype.asm (7 of 7)
